packages/dashboard-backend/src/devworkspace-client/services/api/workspace-api.ts

```diff
@@ -68,8 +68,8 @@ export class DevWorkspaceApi implements IDevWorkspaceApi {
 
   async create(devworkspace: V1alpha2DevWorkspace): Promise<V1alpha2DevWorkspace> {
     try {
-      if (!devworkspace.metadata?.name || !devworkspace.metadata?.namespace) {
-        throw 'DevWorkspace.metadata with name and namespace are required';
+      if (!devworkspace.metadata?.namespace) {
+        throw 'DevWorkspace `metadata.namespace` is required';
       }
```

Comment:
A safer check might be that either `generateName` or `name` is present, as if both are missing the object will be rejected during creation.

Reply:
Thanks! Fixed!